### DISCUSSION TITLE
How to change the name of the virtual webcam from OBS virtual cam to something else

### DISCUSSION THREAD

**Discussion by ghost** [obsproject/obs-studio#4204] [Development Questions] [answered]
How to change the name of the virtual webcam from OBS virtual cam to something else

**Comment by ghost** [comment]
@mufunyo  help, please

**Reply by Heufneutje** [reply]
Pretty sure this has been asked before https://github.com/obsproject/obs-studio/discussions/3938

**Comment by WizardCM** [comment, marked as answer]
Per #3938, this will something we have interest in providing the ability to do.
